data/emulator.js

```diff
@@ -390,8 +390,11 @@ class EmulatorJS {
         if (del){
             document.querySelector('div[class="ejs_ad_iframe"]').remove();
         }else{
-            document.querySelector('iframe[src="'+this.config.adUrl+'"]').src = url;
+            try{
```

Comment:
Can we fix the spacing here.

- functions should always be `function() {`, with a space between the `()` and `{`.
- if statements should always be `if (condition) {` with a space between the `)` and the `{`
- else statements should always be `} else {`, with a space both before and after the `else`
- try statements should always be `try {`, with a space before the `{` and after the `try`
- catch statements should always be `} catch(e) {}`, with a space after the first `{` and a space before the `{}`

This is the coding standard for this project, to make the code look clean and not cluttered due to the lack of spaces

Reply:
Done!